README.md

```diff
@@ -1,6 +1,6 @@
 ![Erda logo](./docs/assets/logo-small.jpg)
 
-## An enterprise-grade application building, deploying, monitoring platform
+## An enterprise-grade Cloud-Native application platform.
```

Comment:
May be no `.` at the end is better ?

Reply:
@Dreamica

Reply:
Agree with @sfwn.
`.` is unnecessary and “cloud-native” should be lowercase.